test/snapshot/assign.test.js

```diff
@@ -46,4 +46,7 @@ describe('assign', () => {
   it('>>=', () => {
     expect(parser.parseEval("$var >>= $var;")).toMatchSnapshot();
   });
+  it('with ref', () => {
+    expect(parser.parseEval("$bar = &$foo;")).toMatchSnapshot();
```

Comment:
does not test positions

Reply:
for position we use other test file, i think we should keep position for all tests, but let's do it in other PR